core/BUILD.bazel

```diff
@@ -18,10 +17,25 @@ java_library(
 )
 
 java_library(
+    name = "inprocess",
+    srcs = glob([
+        "src/main/java/io/grpc/inprocess/*.java",
+    ]),
+    visibility = ["//visibility:public"],
+    deps = [
+        ":internal",
+        ":core",
+        "@com_google_code_findbugs_jsr305//jar",
+        "@com_google_guava//jar",
+    ],
+)
+
+java_library(
     name = "internal",
     srcs = glob([
         "src/main/java/io/grpc/internal/*.java",
     ]),
+    visibility = ["//visibility:public"],
```

Comment:
It seems this should have reduced visibility. I guess everything in grpc-java can use it, though? I guess I don't know how cross-repo visibility works in Bazel...